### DISCUSSION TITLE
Itron OpenWay Electricity meter, how to enter pairing mode

### DISCUSSION THREAD

**Discussion by JoeDaddy7105** [Koenkk/zigbee2mqtt#9166] [Devices]
Had my power meter replaced by my utility (Xcel Energy in Colorado, USA) and when I looked up the new meters I found they are Itron OpenWay AMI7 smart meters which have a ZigBee 2.4GHz radio. I picked up a CC22531 ZigBee dongle to test and see if I can connect to this new meter. I have the CC2531 flashed and zigbee2mqtt setup but I am not exactly sure how to put the meter in to pairing mode. I have also tried sniffing for ZigBee traffic but have not picked anything up. Do I need to pair with my gateway before I can sniff the traffic or should sniffing pickup any broadcast ZigBee messages nearby? I have tried pressing the button to the right of the display on the meter but it does not appear to do anything and no new devices are found. Any thoughts on how to get this meter in to pairing mode.?

Here is my meter for reference
![PXL_20210921_200829423-2](https://user-images.githubusercontent.com/24470441/137558719-e8711e59-e7c0-4e5f-89be-7aab591b1f28.jpg)

Here is some info on these meters: https://www.itron.com/-/media/feature/products/documents/spec-sheet/openway-centron-meter.pdf

**Comment by srpomeroy** [comment]
Did you ever figure this out @JoeDaddy7105 ?

**Comment by PuckleGit** [comment]
I don't think you'll be allowed to enable paring mode as it would be connected to the HAN managed by the utility provider on the install of the meter

I'm UK based but we have smart electrical and gas meters that also have 'ZigBee support' written in their specifications, but from what I can gather it's a version used by utility providers so that they can get the meter usage uploaded to their database and has some extra authorisation process alongside it.

You could try reading the local data through the Optical port on the bottom right with an Optical reader. I've seen some German and Dutch set-ups that read meters that way but outside of those two countries I can't say for sure it'll work on yours, but worth a try

**Reply by srpomeroy** [reply]
Makes sense and can confirm after additional research. There are a handful of devices meant to consume the zigbee HAN network. And you need to have the utility company enable pairing for a specific MAC address. Waiting for some feedback on what devices are known to be supported and what to look for in devices that haven’t gone through their certification process. 

Here are some devices people have had luck with. 

https://shop.emporiaenergy.com/products/utility-connect

https://rainforestautomation.com/us-retail-store/eagle-3-energy-gateway-and-smart-home-hub/
